core/controllers/creator_dashboard.py

```diff
@@ -110,13 +115,38 @@ def get(self):
 class CreatorDashboardPage(base.BaseHandler):
     """Page showing the user's creator dashboard."""
 
+    EDITOR_PAGE_DEPENDENCY_IDS = ['codemirror']
```

Comment:
This is misleading: call it ADDITIONAL_DEPENDENCY_IDS or similar. This isn't the editor page.

Reply:
Done